test/FunctionalTests/Tests/DataOData/Tests/OData.Writer.Tests/Atom/WriterMetadataDocumentAtomTests.cs

```diff
@@ -23,8 +23,9 @@ public class WriterMetadataDocumentAtomTests : ODataWriterTestCase
         [InjectDependency]
         public MetadataWriterTestDescriptor.Settings Settings { get; set; }
 
+        // For comment out test cases, see github: https://github.com/OData/odata.net/issues/883
         [Ignore] // Remove Atom

```

Comment:
Some test cases are mixed with atom and json, Some are not. To distinguish them needs time. So, after QP, we have tasks for it.